src/Monolog/Formatter/HtmlFormatter.php

```diff
@@ -54,12 +54,11 @@ public function __construct(?string $dateFormat = null)
      */
     protected function addRow(string $th, string $td = ' ', bool $escapeTd = true): string
     {
-        $th = htmlspecialchars($th, ENT_NOQUOTES, 'UTF-8');
-        if ($escapeTd) {
-            $td = '<pre>'.htmlspecialchars($td, ENT_NOQUOTES, 'UTF-8').'</pre>';
+       if ($escapeTd) {
+            $td = "<pre>{$this->convertToSpecialChars($td)}</pre>";
         }
 
-        return "<tr style=\"padding: 4px;text-align: left;\">\n<th style=\"vertical-align: top;background: #ccc;color: #000\" width=\"100\">$th:</th>\n<td style=\"padding: 4px;text-align: left;vertical-align: top;background: #eee;color: #000\">".$td."</td>\n</tr>";
+        return "<tr style=\"padding: 4px;text-align: left;\">\n<th style=\"vertical-align: top;background: #ccc;color: #000\" width=\"100\">{$th}:</th>\n<td style=\"padding: 4px;text-align: left;vertical-align: top;background: #eee;color: #000\">{$this->convertToSpecialChars($td)}</td>\n</tr>";
```

Comment:
This change isn't correct. th is not escaped anymore above, and td is escaped once or twice depending on $escapeTd